modules/ocf_backups/manifests/offsite.pp

```diff
@@ -11,6 +11,10 @@
       source => 'puppet:///modules/ocf_backups/upload-to-box',
       mode   => '0755';
 
+    '/opt/share/backups/prune-old-backups':
```

Comment:
we need to add a cronjob for this as well, right?

Reply:
Yeah, gonna do that before merging. Just didn't want to put it in before the code got properly reviewed, to prevent it from running.